docs/api.md

```diff
@@ -4552,4 +4816,50 @@ These are in alphabetic order, with the exception that the `Extended` versions o
 ## Property Detailed (object)
 + name: `first_name` (string, required) - The name of the Property.
 + publishedAt: `2018-01-21T00:04:11.153Z` (string, required) - Publishing timestamp of the form that defined this property for the first time.
-+ forms: (array[Form KeyValue]) - List of forms that create the property
\ No newline at end of file
++ forms: (array[Form KeyValue]) - List of forms that create the property
+
+## EntitySummaryFields (object)
++ uuid: `uuid:85cb9aff-005e-4edd-9739-dc9c1a829c44` (string, required) - The `uuid` of the Entity that uniquely identifies the Entity.
++ datasetName: `people` (string, required) - Name of the dataset
```

Comment:
Do we need to include the dataset name? I don't think that submissions return the `xmlFormId`. Also, the dataset name by itself won't identify the dataset: the project ID is also needed. In other words, I'm thinking that we should return either more information (`projectId`) or less information (`datasetName`).

Reply:
`datasetName` removed